package.json

```diff
@@ -10,6 +10,7 @@
   "repository": "git://github.com/ekalinin/sitemap.js.git",
   "author": "Eugene Kalinin <e.v.kalinin@gmail.com>",
   "dependencies": {
+    "lodash": "^4.17.10",
     "underscore": "^1.7.0",
```

Comment:
should probably remove underscore if you are going to replace it with lodash

Reply:
Ups 🙈 

Good catch, fixed.